README.md

```diff
@@ -102,7 +102,7 @@ With docker compose:
 version: '3'
 services:
   gpodder2go:
-    image: oxtyped/gpodder2go:latest
+    image: ghcr.io/oxtyped/gpodder2go:latest
```

Comment:
Thanks! 🙇 

lgtm, I think we just need to change `latest` to `main` as that's what we're currently tagging the container.

Reply:
Alright, I changed that. It would be best to change it back on the next release though. (main updates on every commit)